.fixtures.yml

```diff
@@ -0,0 +1,8 @@
+fixtures:
+    symlinks:
+      tea:  "#{source_dir}"
+      test: "#{source_dir}/spec/fixtures/test"
+    repositories:
+      stdlib:
```

Comment:
to you mean stdlib? its possible it is not needed i think i just added without thinking